Description: exclude windows dependencies from Cargo.toml
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -107,13 +107,3 @@
     "termios",
     "fs",
 ]
-
-[target."cfg(windows)".dependencies.windows]
-version = "0.58.0"
-features = [
-    "std",
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_Security",
-]
-default-features = false
